sdk/formrecognizer/azure-ai-formrecognizer/src/main/java/com/azure/ai/formrecognizer/models/CopyAuthorization.java

```diff
@@ -92,8 +95,8 @@ public String getAccessToken() {
      *
      * @return the {@code expiresOn} value.
      */
-    public long getExpiresOn() {
-        return this.expiresOn;
+    public OffsetDateTime getExpiresOn() {
```

Comment:
@srnagar Was wondering if something like this could work, where we convert the `long` value to `OffsetDateTime` in the getter only?

As we need the `JsonProperty` for serializing to the long value of `expirationDateTimeTicks` [here](https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/formrecognizer/azure-ai-formrecognizer/src/main/java/com/azure/ai/formrecognizer/models/CopyAuthorization.java#L124-L136)

Reply:
This is an autogenerated class. It shouldn't be modified.

Reply:
From discussion offline, it seems like this is not an autogenerate class. Fix the license header for this file.

Converting long to UTC OffsetDateTime type is fine. Also, fix the javadoc for this method.